### ISSUE TITLE
PHP 7.3 compatibility issue

### ISSUE THREAD

**Issue by thatside** [squizlabs/PHP_CodeSniffer#2319]
PHPCS causes warnings thrown for every file with following error text:
```
STDERR: PHP Warning:  "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"? in /home/thatside/work/project/vendor/squizlabs/php_codesniffer/src/Tokenizers/Tokenizer.php on line 746
STDERR: PHP Stack trace:
STDERR: PHP   1. {main}() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/bin/phpcs:0
STDERR: PHP   2. PHP_CodeSniffer\Runner->runPHPCS() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/bin/phpcs:18
STDERR: PHP   3. PHP_CodeSniffer\Runner->init() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/src/Runner.php:70
STDERR: PHP   4. PHP_CodeSniffer\Ruleset->__construct() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/src/Runner.php:294
STDERR: PHP   5. PHP_CodeSniffer\Ruleset->populateTokenListeners() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/src/Ruleset.php:218
STDERR: PHP   6. PHP_CodeSniffer\Standards\Squiz\Sniffs\Functions\FunctionDeclarationSniff->register() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/src/Ruleset.php:1197
STDERR: PHP   7. PHP_CodeSniffer\Standards\Squiz\Sniffs\Functions\FunctionDeclarationSniff->parse() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/src/Sniffs/AbstractPatternSniff.php:90
STDERR: PHP   8. PHP_CodeSniffer\Standards\Squiz\Sniffs\Functions\FunctionDeclarationSniff->createTokenPattern() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/src/Sniffs/AbstractPatternSniff.php:825
STDERR: PHP   9. spl_autoload_call() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/src/Sniffs/AbstractPatternSniff.php:917
STDERR: PHP  10. PHP_CodeSniffer\Autoload::load() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/src/Sniffs/AbstractPatternSniff.php:917
STDERR: PHP  11. PHP_CodeSniffer\Autoload::loadFile() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/autoload.php:134
STDERR: PHP  12. include() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/autoload.php:167
STDERR: PHP  13. spl_autoload_call() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/src/Tokenizers/PHP.php:14
STDERR: PHP  14. PHP_CodeSniffer\Autoload::load() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/src/Tokenizers/PHP.php:14
STDERR: PHP  15. PHP_CodeSniffer\Autoload::loadFile() /home/thatside/work/project/vendor/squizlabs/php_codesniffer/autoload.php:134
```

**Comment by jrfnl** [comment]
Please upgrade your PHPCS install. This was fixed in both PHPCS 2.9.2 for the 2.x branch, as well as PHPCS 3.3.1 for the 3.x branch.

**Comment by thatside** [comment]
Sorry for bothering you, should have checked this before opening an issue. 
Thanks for your help!
